sql/wsrep_mysqld.cc

```diff
@@ -2283,15 +2283,17 @@ void wsrep_to_isolation_end(THD *thd) {
       (long long)wsrep_thd_trx_seqno(req), wsrep_thd_client_mode_str(req),   \
       wsrep_thd_client_state_str(req), wsrep_thd_transaction_state_str(req), \
       req->get_command(), req->lex->sql_command,                             \
-      (req->rewritten_query.length() ? req->rewritten_query.c_ptr_safe()     \
-                                     : req->query().str),                    \
+      (req->rewritten_query().length()                                       \
+                ? const_cast<String&>(req->rewritten_query()).c_ptr_safe()   \
+                : req->query().str),                    \
```

Comment:
Formatting. Have you ran clang-format on this file? That `\` should go to the same column as the others.

Reply:
wsrep_thd_rewritten_query()...